src/components/nav-bar/nav-bar.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
```

Comment:
Is the file extension correct? Do you use typescript in the project?

Reply:
I'm not sure, some components have prop types, so, I think so. But, in this case, a normal import is fine, isn't it?
Import React from react;